src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -175,6 +175,31 @@
         it_behaves_like 'a full diff requested for', 'bigfile_archive.tar.gz/bigfile.txt'
       end
     end
+
+    context 'with :diff_to_superseded set' do
+      let(:superseded_bs_request) { create(:bs_request) }
+      context 'and the superseded request is superseded' do
+        before do
+          bs_request_submit_action
```

Comment:
What's that?

Reply:
Because it is lazy evaluated, but I guess we can drop it! c&p from previous test.

Reply:
at least not from stack overflow :nerd_face: